Signal/test/view controllers/Signals/TSMessageAdapters/TSMessageAdapterTest.m

```diff
@@ -206,13 +214,27 @@ - (void)testPerformCopyEditingActionWithPhotoMessage
     UIPasteboard.generalPasteboard.items = @[];
     XCTAssertNil(UIPasteboard.generalPasteboard.image);
 
-    // Grab some random existing image
-    UIImage *image = [UIImage imageNamed:@"savephoto"];
-    TSPhotoAdapter *photoAdapter = [[TSPhotoAdapter alloc] initWithImage:image];
+    NSError *error;
+    TSAttachmentStream *attachment = [[TSAttachmentStream alloc] initWithContentType:@"image/jpeg"];
+    [attachment writeData:self.fakeAudioData error:&error];
```

Comment:
We now assert that the attachment has a real filepath, which broke this lazy test.